src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -127,6 +96,42 @@ static bool CallchainIsInUserSpaceInstrumentation(
       });
 }
 
+void UprobesUnwindingVisitor::SendFullAddressInfoToListener(
+    TracerListener* listener, const unwindstack::FrameData& libunwindstack_frame) {
```

Comment:
You should use `listener_` instead of passing it now that it's a member function.

Reply:
Done.